### DISCUSSION TITLE
Will vaccinations by ages and vaccinations by manufacturers not be put in new ETL process?

### DISCUSSION THREAD

**Discussion by ElsieZhang** [owid/covid-19-data#2843] [General]
I noticed that you decided to update data in ETL instead of Github. However, not all datasets are included in ETL, like vaccinations by ages and vaccinations by manufacturers. Does this mean those datasets that are not migrated into ETL will be discontinued?

**Comment by lucasrodes** [comment]
Hi @ElsieZhang! 

Thanks for your interest in the project. The data is in ETL, and I've now added the links to our documentation. These should be live in the next few minutes.

One thing to note is that the data for vaccinations by age and manufacturer won't be updated anymore.

We are working on an article to clarify all of this and better explain what's being updated regularly and what's not.
